### ISSUE TITLE
Honestly just let the regular nnUNet training run and pick model_final_checkpoint.model. Only consider using model_best.model when you see the green line in progress.ong to drop again. model_best.model uses an exponential moving average of the green line to determine what is considered 'best'. This is not very well tunes because I don't use it and it should also never be used to report 5-fold cross-validation scores (overfitting to validation set).

### ISSUE THREAD

**Issue by peeelS** [MIC-DKFZ/nnUNet#1205]
Honestly just let the regular nnUNet training run and pick model_final_checkpoint.model. Only consider using model_best.model when you see the green line in progress.ong to drop again. model_best.model uses an exponential moving average of the green line to determine what is considered 'best'. This is not very well tunes because I don't use it and it should also never be used to report 5-fold cross-validation scores (overfitting to validation set).
Please only modify the training length once you completed a full training and truly believe that shorter training could be beneficial. But it's probably even better to just modify the checkpointing behavior and save a separate checkpoint file every XXX epochs
Best,
Fabian

_Originally posted by @FabianIsensee in https://github.com/MIC-DKFZ/nnUNet/issues/695#issuecomment-1011331359_
